tests/test_gluu_kubedb.py

```diff
@@ -0,0 +1,136 @@
+import pygluu.kubernetes.kubedb as module0
```

Comment:
The use of KubeDB has been depreciated. We will start using Bitnamis helm chart for installing Redis / Postgres. This file and all KubeDB mentions can be removed.